behaviour_and_protocol/consulting_101.md

```diff
@@ -1,26 +1,95 @@
-- When estimating, your job is to be realistic. A common error among inexperienced consultants is to make the client believe that the task is easy and therefore making them happy. This leads to going over budget and over the defined deadline. Share your concerns so that the client can identify what are their possibilities. Help them brainstorm to overcome the possible problems. Sales should be informed when your concerns might have larger implications, and you are working with a potential client.
-
-## Etiquette
-- Arrive on time and be prepared.
-- Remember time tracking in the correct project.
-- When you call a meeting, always have an agenda, and for longer meetings, make sure you take notes.
-- Always listen but don't be afraid to take the lead.
-- Dress appropriately whether the meeting is in-person or handled remotely, but remember that you can be professional without wearing formal attire.
-- For remote meetings, have the camera on, make sure that the background is presentable, and conduct the meeting in a quiet room with no disturbances. Mute yourself if there is any background noise or disturbances.
-- Remember to be aware of your digital presence. Remain professional throughout the process whether this means in Asana, Git, your Harvest comments, or the codebase.
+Consulting 101
+==============
+
+Introduction to Consulting
+--------------------------
+
+As a consulting company, we are fortunate to have collaborated with a wide range of companies over the years from solo entrepreneurs to large Danish corporations. Key to this success is that we acknowledge that every client is different and has different needs.
+
+If you are coming from a product company, it can be unclear what consulting is, and what your new role as a consultant requires. Our goal with this document is to provide a clearer overview of consulting. We believe that training people and bringing them closer to the client ensures that we can give clients better advice and products. 
+
+As a reminder, please note that the following are guidelines and not strict rules.
+
+This document is divided into the following sections:
+- [Client Communication](#client-communication)
+- [Technical Communication](#technical-communication)
+- [Etiquette and Professionalism](#etiquette-and-professionalism)
+- [Proactiveness](#proactiveness)
+- [Solution-Oriented Approach](#solution-oriented-approach)
+- [Understanding the Client’s Perspective](#understanding-the-clients-perspective)
+- [Client-Specific Insights](#client-specific-insights)
+
+Client Communication
+--------------------
+
+* **Recognize and understand the client's preferred communication style.**
+  * If a client prefers to understand the context for your solutions in detail, provide comprehensive explanations.
+* **Keep communication transparent and public.**
+  * When communicating about a specific task, be sure to keep the communication in a shared platform like Asana or Slack. Ideally on the task itself.
```

Comment:
Swapping Asana and Slack would make it more congruent with the following sentence.